chore: Setup Verified - Added Contributor Details 🎮

adityashirsatrao007 opened the pull request:
Issue : #27

# Description
Successfully installed Unity **6000.3.2f1** and opened the PixDash project.

### Changes
- Added \contributors/adityashirsatrao007.txt\ with details.
- Added \contributors/adityashirsatrao007.png\ as verification proof of the setup (screenshot).

Ready to build!

aadyagupta23 commented:
Add a drive link to your screenshot in <your_roll_number>.txt, dont upload a seperate png. If you have any doubts, ask away.

adityashirsatrao007 commented:
Updated per guidance:
pwsh
1. Removed separate PNG.
2. Added Drive link to contributors/adityashirsatrao007.txt.

(Note: I'm combining the 'Contributor Details' and 'Setup' into this single PR for efficiency, as this is my entry PR.)
